src/main/webapp/view/discussion/discussion.js

```diff
@@ -256,7 +175,10 @@ class DiscussionComment extends HTMLElement {
     };
     this.shadowRoot.querySelector(SELECTOR_POST_REPLY).onclick = () => {
       const textarea = this.shadowRoot.querySelector(SELECTOR_REPLY_TEXTAREA);
-      postReply(textarea, this.comment.commentKey.id);
+      this.#manager.postReply(textarea.value, this.comment.commentKey.id)
+          .then(() => {
+            loadDiscussion();
+          });
```

Comment:
Work in constructor? Is postReply communicating with the server some how?

Reply:
This is still setting up the functionality of the shadow DOM.  It just registers the event listeners, doesn't actually call them.

Reply:
This particular section says:
When the POST_REPLY button in the shadowRoot is pressed, post a reply then reload the discussion.

Reply:
It still seems like work as I look through what postReply() does, I see that postComment() does some form of for loop or so. Maybe we can have an initialize() function here

Reply:
We chatted and I made the suggested changes in #264 .  Added a TODO here